src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -283,16 +283,18 @@ private void createElevatorEdgesAndAddThemToGraph(
             levels = Math.abs(fromVertexLevelIndex - toVertexLevelIndex);
         }
 
+        String elevatorLabel = fromVertex.getLabel() + "-" + toVertex.getLabel();
+
```

Comment:
As these are used from GTFS source, and they already include the pathway id in the label, there is no risk of collision from these.